stuffer/s2n_stuffer.h

```diff
@@ -36,7 +42,9 @@ struct s2n_stuffer {
     /* Is this stuffer growable? */
     unsigned int growable:1;
 
-    /* A growable stuffer can also be temporarily tainted */
+    /* Can this stuffer be safely resized?
+     * A growable stuffer can be temporarily tainted by a raw read/write,
+     * preventing it from resizing. */
     unsigned int tainted:1;
```

Comment:
Slightly confused by the comment/naming combo. Is '1' yes, this stuffer can be safely resized? or yes, this stuffer is tainted?